### PR TITLE
feat: write README and expose functions

### LINKED ISSUE #8: fill the readme
- Add a small description of the package before "Installation" (and copy it to the package.json's description field)
- Update the example in Usage to something "real" that shows how to use the library (short not too simple)
- Add a "Documentation" title (before "License") that links to the built documentation at https://zakodium.github.io/is-my-file-ready
- update CI svg link to "main" instead of "master"

### LINKED ISSUE #9: export the check functions from the package

### PR DESCRIPTION
**Pull request by Xstoudi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,8 @@
 [![Test coverage][codecov-image]][codecov-url]
 [![npm download][download-image]][download-url]
 
-.
+# Description
```
Comment (targos) [inline README.md]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -14,19 +15,41 @@
 ## Usage
 
 ```js
-import { myModule } from 'is-my-file-ready';
-
-const result = myModule(args);
-// result is ...
+const result = await isMyFileReady('./myfile.txt', [
```
Comment (targos) [inline README.md:22]: Add the imports above

File: README.md [hunk 3]
```diff
@@ -14,19 +14,43 @@
...
 ```js
-import { myModule } from 'is-my-file-ready';
-
-const result = myModule(args);
-// result is ...
+import isMyFileReady, { sameSize, endsWithStr } from 'is-my-file-ready';
+
+const result = await isMyFileReady('./myfile.txt', [
+  sameSize(expectedSize),
+  endsWithStr(expectedEnds),
```
Comment (targos) [inline README.md:24]: these variables are undefined

Comment (targos) [reply README.md:24]: they are still undefined

Comment (Xstoudi) [reply README.md:24]: Forgot to push x)

**Pull request merged**